Source/Core/CheckoutAPIService.swift

```diff
@@ -14,6 +14,7 @@ protocol CheckoutAPIProtocol {
     var logger: FramesEventLogging { get }
     init(publicKey: String, environment: Environment)
     func createToken(_ paymentSource: PaymentSource, completion: @escaping (Result<TokenDetails, TokenisationError.TokenRequest>) -> Void)
+    func createSecurityCodeToken(securityCode: String, completion: @escaping (Result<SecurityCodeTokenDetails, TokenisationError.SecurityCodeError>) -> Void)
```

Comment:
formatting looks a bit odd here as well and on line 40 below. I will wait for you to confirm before I add more commets which are similar.

Reply:
can you elaborate on the oddness please? @deepesh-vasthimal-cko

Reply:
<img width="1501" alt="Screenshot 2023-10-31 at 12 57 26" src="https://github.com/checkout/frames-ios/assets/125963311/0f5431cf-e209-4c57-9ad9-6d043bde6f15">

it looks like this in my Xcode @deepesh-vasthimal-cko